### ISSUE TITLE
Error: Cannot find module 'zombie'

### ISSUE THREAD

**Issue by jhedstrom** [phase2/grunt-drupal-tasks#175]
This is using a default install, setup via `yo gadget`.
```
  @api @javascript
  Scenario: Ensure the Reports page is loaded into an overlay when the Reports   # features/example.feature:20
    menu item is clicked.
    Given I am logged in as a user with the "administrator" role                 # Drupal\DrupalExtension\Context\DrupalContext::assertAuthenticatedByRole()
      Server process has been terminated: (8) [
      module.js:340
          throw err;
                ^
      Error: Cannot find module 'zombie'
          at Function.Module._resolveFilename (module.js:338:15)
          at Function.Module._load (module.js:280:25)
          at Module.require (module.js:364:17)
          at require (module.js:380:17)
          at Object.<anonymous> (/private/var/folders/ht/d5mpqc8j7ljblc_2xzbm3m400000gn/T/mink_nodejs_serverBjAkpy:2:16)
          at Module._compile (module.js:456:26)
          at Object.Module._extensions..js (module.js:474:10)
          at Module.load (module.js:356:32)
          at Function.Module._load (module.js:312:12)
          at Function.Module.runMain (module.js:497:10)
      ] (RuntimeException)
```

**Comment by peterschuelke** [comment]
I got the same error

**Comment by peterschuelke** [comment]
By adding 'export NODE_PATH="/usr/local/lib/node_modules"' to the .bash_profile this error was exchanged for:

  @api @javascript
  Scenario: Ensure the Reports page is loaded into an overlay when the Reports # features/example.feature:20
    menu item is clicked.
    Given I am logged in as a user with the "administrator" role               # Drupal\DrupalExtension\Context\DrupalContext::assertAuthenticatedByRole()
      Server process has been terminated: (1) [/usr/local/lib/node_modules/zombie/node_modules/jsdom/lib/jsdom/living/node-iterator.js:3
      const defineGetter = require("../utils").defineGetter;
      ^^^^^
      SyntaxError: Use of const in strict mode.
          at exports.runInThisContext (vm.js:73:16)
          at Module._compile (module.js:443:25)
          at Object.Module._extensions..js (module.js:478:10)
          at Module.load (module.js:355:32)
          at Function.Module._load (module.js:310:12)
          at Module.require (module.js:365:17)
          at require (module.js:384:17)
          at Object.<anonymous> (/usr/local/lib/node_modules/zombie/node_modules/jsdom/lib/jsdom/living/index.js:22:1)
          at Module._compile (module.js:460:26)
          at Object.Module._extensions..js (module.js:478:10)
          at Module.load (module.js:355:32)
          at Function.Module._load (module.js:310:12)
          at Module.require (module.js:365:17)
          at require (module.js:384:17)
          at Object.<anonymous> (/usr/local/lib/node_modules/zombie/lib/dom/index.js:5:11)
          at Module._compile (module.js:460:26)
      ] (RuntimeException)

**Comment by jhedstrom** [comment]
Note the latest zombie.js doesn't appear to work with the mink zombie driver: https://github.com/minkphp/MinkZombieDriver/issues/132

**Comment by grayside** [comment]
Potentially related to #125, or at least another sign that we consider something like #247

**Comment by arithmetric** [comment]
This should be fixed in v1.0.0-alpha1 and later due to upgrading MinkZombieDriver and zombie.js.
